projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-feed-locator.php

```diff
@@ -49,7 +49,7 @@ private function is_podcast_feed( $file ) {
 	private function safely_load_xml( $xml ) {
 		$disable_entity_loader = PHP_VERSION_ID < 80000;
 
-		if ( $disable_entity_loader && ! function_exists( 'libxml_disable_entity_loader' ) ) {
+		if ( $disable_entity_loader ) {
```

Comment:
I approved and then noticed the failing tests, seemingly caused by that:
https://github.com/Automattic/jetpack/actions/runs/6937070982/job/18870424528?pr=34222